scripts/faster_execution.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# This script greatly speeds up execution of qutebrowser commands when a
+# qutebrowser instance already exists.  Call it in place of your system's
+# qutebrowser executable.  It is based on the following work:
+#
+# scripts/open_url_in_instance.sh
```

Comment:
I don't think that shell script is still needed now.

Reply:
Done.